frontend/src/pages/acceleratorProfiles/screens/manage/ManageAcceleratorProfile.tsx

```diff
@@ -36,6 +40,14 @@ const ManageAcceleratorProfile: React.FC<ManageAcceleratorProfileProps> = ({
     }
   }, [existingAcceleratorProfile, setState]);
 
+  React.useEffect(() => {
+    if (!existingAcceleratorProfile) {
+      setState('name', profileNameDesc.k8sName.value);
+    }
+    setState('displayName', profileNameDesc.name);
+    setState('description', profileNameDesc.description);
+  }, [existingAcceleratorProfile, profileNameDesc, setState]);
```

Comment:
I'd prefer to update `useK8sNameDescriptionFieldData` to accept initial values that can be supplied instead of work around updating state in a `useEffect`.

Then you also wouldn't need to change `getDescriptionFromK8sResource` or `getDisplayNameFromK8sResource` either.